### PR TITLE
fix some errmsg report,in framework/ir/, about 21 files

### PR DESCRIPTION
**Pull request by wanghuancoder**
### PR types

Others
### PR changes

Others
### Describe

fix some errmsg report,in framework/ir/, about 21 files

### CODE REVIEW

File: paddle/fluid/framework/ir/fuse_elewise_add_act_pass.cc [hunk 1]
```diff
@@ -347,8 +356,9 @@ std::vector<Node *> FuseElewiseAddActPass::ReplaceNode(
                    }
                    return node;
                  });
-  PADDLE_ENFORCE(has_replaced, "Not find %s in the node list.",
-                 cur_node->Name());
+  PADDLE_ENFORCE_EQ(has_replaced, true,
+                    platform::errors::NotFound("Not find %s in the node list.",
```
Comment (chenwhql) [inline paddle/fluid/framework/ir/fuse_elewise_add_act_pass.cc]: found

File: paddle/fluid/framework/ir/fuse_pass_base.cc [hunk 2]
```diff
@@ -25,14 +25,19 @@ void FusePassBase::Init(const std::string& repr, Graph* graph) const {
...
 }
 
 void FusePassBase::AddStatis(int count_of_fused) const {
-  PADDLE_ENFORCE(graph_);
-  PADDLE_ENFORCE(!repr_.empty());
+  PADDLE_ENFORCE_NOT_NULL(
+      graph_, platform::errors::InvalidArgument("Graph cannot be nullptr."));
+  PADDLE_ENFORCE_EQ(
+      repr_.empty(), false,
+      platform::errors::InvalidArgument("Fuse pass must init with a name."));
```
Comment (chenwhql) [inline paddle/fluid/framework/ir/fuse_pass_base.cc]: must be initialized with

File: paddle/fluid/framework/ir/graph_helper.cc [hunk 3]
```diff
@@ -136,7 +136,9 @@ bool FindCircleSubGraph(const Graph &graph,
 std::vector<ir::Node *> TopologySortOperations(const Graph &graph) {
   std::map<ir::Node *, std::set<ir::Node *, ir::NodeComp>, ir::NodeComp>
       adj_list = BuildOperationAdjList(graph);
-  PADDLE_ENFORCE(!HasCircleInternal(adj_list, nullptr));
+  PADDLE_ENFORCE_EQ(HasCircleInternal(adj_list, nullptr), false,
+                    platform::errors::InvalidArgument(
+                        "Generated graph shouldn't have cycle."));
```
Comment (chenwhql) [inline paddle/fluid/framework/ir/graph_helper.cc]: have -> contain 好一些

File: paddle/fluid/framework/ir/fuse_elewise_add_act_pass.cc (hunk 1 above)
Comment (wanghuancoder) [reply paddle/fluid/framework/ir/fuse_elewise_add_act_pass.cc]: done,thx.

File: paddle/fluid/framework/ir/fuse_pass_base.cc (hunk 2 above)
Comment (wanghuancoder) [reply paddle/fluid/framework/ir/fuse_pass_base.cc]: done,thx.

File: paddle/fluid/framework/ir/graph_helper.cc (hunk 3 above)
Comment (wanghuancoder) [reply paddle/fluid/framework/ir/graph_helper.cc]: done,thx.

**Review by luotao1 — APPROVED** [review]
LGTM

**Review by raindrops2sea — APPROVED** [review]
LGTM

**Pull request merged**